package.json

```diff
@@ -15,7 +15,7 @@
   "main": "index.js",
   "scripts": {
     "lint": "eslint .",
-    "test": "mocha --UV_THREADPOOL_SIZE=100 --require test/init/init.js test/*.js",
+    "test": "mocha test/*.test.js node_modules/juggler-v3/test.js node_modules/juggler-v4/test.js",
```

Comment:
do we not need the `UV_THREADPOOL_SIZE` flag anymore?

Reply:
Let me add it back; I removed it by accident. Thanks!